modules/microshift-restoring-data-backups.adoc

```diff
@@ -4,42 +4,50 @@
 
 :_content-type: PROCEDURE
 [id="microshift-restoring-data-backups-manually_{context}"]
-= Restoring {product-title} data backups manually
+= Restoring {microshift-short} data backups manually
 
-You can restore {product-title} data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When you restore a backup, you must use the entire file path.
+You can restore {microshift-short} data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Automated backups are in the `/var/lib/microshift-backups` directory by default. You can use this directory for manually backing up and restoring data by specifying it in each command. When you restore a backup, you must use the entire file path.
 
 [NOTE]
 ====
-On an `rpm-ostree` system, {product-title} backs up and restores data automatically.
+On an `rpm-ostree` system, {microshift-short} backs up and restores data automatically.
 ====
 
 .Prerequisites
+
 * Root access to the host.
 * Full path of the data backup file.
-* {product-title} is stopped.
+* {microshift-short} is stopped.
 
 .Procedure
 
-. Manually restore an application backup file by using the full file path of the backup you want to restore by running the following command:
+. Manually restore {microshift-short} data by using the full file path of the backup you want to restore by running the following command:
 +
 [source,terminal]
 ----
-$ sudo microshift restore /var/lib/<custom_backups>/<my_backup>
+$ sudo microshift restore /var/lib/microshift-backups/<my_manual_backup>
```

Comment:
One small nit: user might restore auto-backup (on ostree), so `my_manual_backup` might not apply too well here, but I'm not sure if we need to split the hair?

Reply:
@pmtk It's a good note. Added as optional. PTAL.